translations/en.json

```diff
@@ -696,6 +696,10 @@
   "newScan": "New Scan",
   "newScanStarted": "New scan started, Click here to show",
   "next": "Next",
+  "nistsp80053": "NIST SP 800-53",
+  "nistsp800171":"NIST SP 800-171",
+  "nistsp80053Expansion": "NIST SP 800-53",
```

Comment:
The `nistsp80053` and `nistsp80053Expansion` have the same translation value. Is there a reason why it's duplicated?

Reply:
yeah I don't think it is required for now